src/main/java/io/vertx/core/http/HttpServerOptions.java

```diff
@@ -20,6 +20,7 @@
 import io.vertx.core.buffer.Buffer;
 import io.vertx.core.json.JsonObject;
 import io.vertx.core.net.*;
+import io.vertx.core.net.impl.SSLHelper;
```

Comment:
SSLHelper is not part of the public API (notice the "impl" package) -we can't expose private implementation classes in the API.